### DISCUSSION TITLE
game crashes, reason: "REI Internals have not been initialized!"

### DISCUSSION THREAD

**Discussion by X3N032** [shedaniel/RoughlyEnoughItems#1291] [Q & A] [answered]
I threw together a bunch of mods to play with some friends, but REI is crashing it on startup. Sorry in advance if this is a very simple problem, but I couldn't find someone else with the fix.

crash log:
[crash-2022-12-22_21.32.17-client.txt](https://github.com/shedaniel/RoughlyEnoughItems/files/10291633/crash-2022-12-22_21.32.17-client.txt)

**Comment by shedaniel** [comment, marked as answer]
Remove MoreOverlays
